packages/patternfly-4/react-core/src/layouts/Page/Page.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/layouts/Page/page.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { BarsIcon } from '@patternfly/react-icons';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+
+export const PageLayouts = {
+  vertical: 'vertical',
+  horizontal: 'horizontal'
+};
+
+const propTypes = {
+  /** Content rendered inside the Page layout */
+  children: PropTypes.node,
+  /** Additional classes added to the Page layout */
+  className: PropTypes.string,
+  /** Page layout type */
+  layout: PropTypes.oneOf(Object.values(PageLayouts)),
+  /** Component to render the background image (e.g. <BackgroundImage />) */
+  background: PropTypes.node,
+  /** Component to render the logo/brand (e.g. <Brand />) */
+  logo: PropTypes.node,
+  /** Component to render the navigation (e.g. <Nav /> */
+  nav: PropTypes.node,
+  /** Vertical layout only: True if the nav is expanded  */
+  isNavOpen: PropTypes.bool,
+  /** Vertical page layout only: Callback function to handle the navigation toggle button */
+  onNavToggle: PropTypes.func,
+  /** Component to render the toolbar (e.g. <Toolbar />) */
+  toolbar: PropTypes.node,
+  /** Component to render the avatar (e.g. <Avatar /> */
+  avatar: PropTypes.node
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  layout: 'vertical',
+  background: null,
+  logo: 'Logo',
+  nav: 'Nav',
+  onNavToggle: () => undefined,
+  isNavOpen: true,
+  toolbar: 'Toolbar',
+  avatar: null
+};
+
+const Page = ({
+  className,
+  children,
+  layout,
+  background,
+  logo,
+  nav,
+  isNavOpen,
+  onNavToggle,
+  toolbar,
+  avatar,
+  ...props
+}) => (
+    <React.Fragment>
+      {background}
```

Comment:
I don't think the background belongs in the layout component (that is what is shown in PF-next).  I think when you build the demo, you can add the backgroynd there.

Reply:
Sounds good!